BitwardenShared/Core/Vault/Services/VaultTimeoutService.swift

```diff
@@ -171,12 +200,43 @@ class DefaultVaultTimeoutService: VaultTimeoutService {
         }
     }
 
+    func sessionTimeoutAction(userId: String?) async throws -> SessionTimeoutAction {
+        let hasMasterPassword = try await stateService.getUserHasMasterPassword(userId: userId)
+        let timeoutAction = try await stateService.getTimeoutAction(userId: userId)
+        guard hasMasterPassword else {
+            let isBiometricsEnabled = try await biometricsRepository.getBiometricUnlockStatus().isEnabled
+            let isPinEnabled = try await isPinUnlockAvailable(userId: userId)
```

Comment:
⛏️ There's only one test for throwing behavior based on `stateService.getUserHasMasterPassword`, but there are 3 tests missing. 
When:
- `try await stateService.getTimeoutAction(...)` throw
- `try await biometricsRepository.getBiometricUnlockStatus().isEnabled` throw
- `try await isPinUnlockAvailable(userId: userId)` throw